@ecomplus/storefront-twbs/scss/presets/_colors.scss

```diff
@@ -28,6 +28,38 @@ $color-shift-weights: map-merge(
   @return $min-shift-weight;
 }
 
+// scss-docs-start color-functions
+// Tint a color: mix a color with white
+@function tint-color($color, $weight) {
+  @if $weight < 0 {
+    @return shade-color($color, -$weight);
+  }
+  @if type-of($color) == color {
+    @return mix(white, $color, $weight);
+  } @else {
+    @return str-insert($color, -#{label-color-weight(-$weight)}, -2);
```

Comment:
Checout se o valor desse `$weight` é o mesmo do que era passado no BS4 na função relativa (`lighten` nesse caso)? Se não for o mesmo isso não deve funcionar dessa forma, deve ter que fazer adaptação por aí ... 😬